app/utils/answersHelpers/ConsignmentAnswersHelper.scala

```diff
@@ -216,7 +221,7 @@ class ConsignmentAnswersHelper(userAnswers: UserAnswers)(implicit messages: Mess
         AccordionSection(
           sectionTitle = Some(messages("unloadingFindings.subsections.incidents", incidentIndex.display)),
           rows = rows,
-          children = endorsementSection +: helper.incidentTransportEquipments
+          children = endorsementSection +: helper.incidentTransportEquipments :+ transhipment
```

Comment:
Maybe
```
children = Seq(
  Seq(endorsementSection),
  helper.incidentTransportEquipments,
  Seq(transhipment)
).flatten
```
for the sake of consistency?

Reply:
yeah I am happy with that, I think this is more performant than the prepend append operations